code/_onclick/click.dm

```diff
@@ -32,11 +32,41 @@
 		usr.MouseWheelOn(src, delta_x, delta_y, params)
 
 
-/client/Click(object, location, control, params)
-	if(!control)
```

Comment:
check if this lets you use .click macros again

Reply:
Oof /tg/ doesn't allow macros so mistakes were made.